src/pika_slot.cc

```diff
@@ -136,9 +134,41 @@ static int doMigrate(pink::PinkCli *cli, std::string send_str){
     slash::Status s;
     s = cli->Send(&send_str);
     if (!s.ok()) {
-        LOG(WARNING) << "slotKvMigrate Send error: " <<strerror(errno);
+        LOG(WARNING) << "Slot Migrate Send error: " <<strerror(errno);
         return -1;
     }
+    // Recv
+    s = cli->Recv(NULL);
```

Comment:
this may bring down performance.
but considering about the condition that large keys are sliced into multiple pieces(batches for migration), it might be the clearest and simplest solution to avoid the possibility of leaving garbage data in socket buffers.

Reply:
found a way works well when large keys are sliced. see https://github.com/Qihoo360/pika/pull/236